### ISSUE TITLE
Versioning

### ISSUE THREAD

**Issue by thomasfedb** [carrierwaveuploader/carrierwave-i18n#3]
How are we intending to indicate what version of `carrierwave-i18n` should be used with any given version of `carrierwave`?

**Comment by araccaine** [comment]
Maybe one could use versioning like [`rails-i18n`](https://github.com/svenfuchs/rails-i18n#gem-installation) does?

They publish the gem with the same major version as rails/railties to indicate the dependency. In their [`rails-i18n.gemspec`](https://github.com/svenfuchs/rails-i18n/blob/11fbfaa1f60d719e645f0cab71cda3f34ee0e971/rails-i18n.gemspec#L19) they set the dependency as well.

Maybe `carrierwave-i18n` can do it also like this:

```ruby
Gem::Specification.new do |s|
  # ...
  s.add_runtime_dependency('carrierwave', '>= 3.0.0', '< 4')
 # ...
end
```

**Comment by mshibuya** [comment]
Resolved by #27.
